setenv.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script aliases the docker cli if the environment variable USE_PODMAN is set to true.
+
+# default value is false if USE_PODMAN is unset or null
+podman=${USE_PODMAN:-false}
+if [ ${podman} == true ]; then
+  alias docker=podman
+  echo "setting alias docker=podman"
+fi
```

Comment:
Why not inline this?

Reply:
I do have the same question, why using a separate script just for setting alias?

Reply:
The reasoning for having it in a separate script was only because I was going off of prior work done [here](https://github.com/devfile/registry-support) by someone else. That repo utilizes the same `setenv.sh` script being used alongside an env variable to toggle between podman or docker.

If it's preferred to be inline that change can definitely be made.

Reply:
The way I've usually done it in other repos is to just use a variable, e.g.
```
DOCKER=docker
if [ <use podman condition> ]; then
  DOCKER=podman
fi

$DOCKER build -t <image> -f <file> <context>
```
this also avoids the need for setting `shopt -s expand_alias`.

Reply:
In my last commit I added the setting of the alias inline so we don't need another script to be doing that work. I initially tried to go with your way @amisevsk with the use of a variable but found that when it was trying to call the `./build.sh` script it was failing with a seg fault, this was not happening with setting the alias. I'm not sure why so more investigation will have to be done into that if we want to not use an alias and rather a variable.